src/commands/build-image.yml

```diff
@@ -77,7 +89,12 @@ parameters:
 
 steps:
   - when:
-      condition: <<parameters.skip-when-tags-exist>>
+      condition: <<parameters.checkout>>
+      steps:
+        - checkout
+
+  - when:
+      condition: <<parameters.ecr-login>> or <<parameters.skip-when-tags-exist>>
```

Comment:
Oh, I didn't actually check if this syntax is legit, this was just a guess!

Reply:
Ok based on https://circleci.com/docs/2.0/configuration-reference/#logic-statements, I think it is like this:

```yaml
condition:
  or:
    - expression_a
    - expression_b
```

I have updated the spec now!